mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class ItalianLinguisticAcceptabilityClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="Itacola",
+        dataset={
+            "path": "gsarti/itacola",
+            "revision": "f8f98e5c4d3059cf1a00c8eb3d70aa271423f636",
+        },
+        description="An Italian Corpus of Linguistic Acceptability taken from linguistic literature with a binary annotation made by the original authors themselves.",
+        reference="https://aclanthology.org/2021.findings-emnlp.250/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["ita-Latn"],
+        main_score="accuracy",
+        date=None,
```

Comment:
Should be a range: (start, end)

Reply:
I have the publication date of course but couldn't find any reference to a data collection period, happy to follow through if there's a suggested guideline for these situations

Reply:
Create a (from, to) for the publication year then (e.g. if it is 2021: `("2021-01-01", "2022-12-31")`